db/seeds.rb

```diff
@@ -5,3 +5,25 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+puts "cleaning database..."
+User.destroy_all
+
+puts "Seeding database with Plutonians..."
+password = "123456"
+
+ken = User.create(email: 'ken@gmail.com', password: password, reset_password_token: "ken", username: 'Ken')
```

Comment:
I don't know if this is on purpose but the usernames for Ken and Jess are capital and tyler and nicole and lowercase. Probably be best to make them all match to avoid future confusion!
Other than that LGTM

Reply:
I also noticed I have interchanging ' and ". I will make those clean and also get rid of the commented out seed instructions when I get back home.